recipes/omas/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "omas" %}
+{% set version = "0.2.6" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "a3f315f4b192d665b96e4f041ce2fc7fd88a43ada9ab183249f5f5d7f54d7f6c" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+  patches:
+    - setup.py.patch
+
+build:
+  number: 0
+  script: "python setup.py install --single-version-externally-managed --record record.txt"
+  noarch: python
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - netcdf4
+    - boto3
+    - botocore 1.7.*
+
+about:
+  home: https://gafusion.github.io/omas
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
```

Comment:
Please raise an issue upstream about including the `license_file` in the source distribution and put a link to the issue (or PR) as a comment in this recipe as a reminder to look for its inclusion in future releases.